rocketmq-spring-boot-starter/src/main/java/org/apache/rocketmq/spring/starter/RocketMQAutoConfiguration.java

```diff
@@ -59,11 +65,11 @@
 @Order
 @Slf4j
 public class RocketMQAutoConfiguration {
-
     @Bean
     @ConditionalOnClass(DefaultMQProducer.class)
     @ConditionalOnMissingBean(DefaultMQProducer.class)
     @ConditionalOnProperty(prefix = "spring.rocketmq", value = {"nameServer", "producer.group"})
+    @Order(1)
```

Comment:
Why interceptor order must be pointed  out?

Reply:
It need to control the beans loaded order, like this:
DefaultMQProducer Order1 > RocketMQTemplate Order2 > TransactionHandlerRegistry Order3

All of these 3 beans are defined in the file  RocketMQAutoConfiguration.java